src/configureStore.js

```diff
@@ -5,22 +5,11 @@ import thunk from 'redux-thunk';
 import history from './history';
 import { CALL_HISTORY_METHOD } from './state/routerActions';
 import { REPORT_ERROR } from './state/reportError';
+import throttle from 'lodash/throttle';
```

Comment:
Interesting. I never used `lodash` before. It is even possible to install throttle only: https://www.npmjs.com/package/lodash.throttle
I am not sure whether it applies to our case though.

Reply:
Yes it's possible, although it's the same, importing `lodash/throttle` only adds the `throttle` function. And we have the `lodash` package in case we need another function from there. I think I also used `isEqual` in a different place.

Reply:
Good to know. Thanks!